pom.xml

```diff
@@ -467,7 +467,7 @@
               <properties>
                 <property>
                   <name>log4j.configurationFile</name>
-                  <value>${project.baseUri}src/main/resources/log4j2-test.xml</value>
+                  <value>${project.baseUri}src/main/resources/log4j2.xml</value>
```

Comment:
Does this mean that there is a single `log4j2` configuration file for both production and tests?

If so, why is the `log4j2-test.properties` file needed?

Reply:
I changed it back to log4j2-test-I was having problems getting it to run with the testfile earlier in the conversion, I just forget to change it back.  I've removed the properties file.